WooCommerce/Classes/ViewRelated/Dashboard/Reviews/ReviewsDashboardCard.swift

```diff
@@ -141,26 +146,24 @@ private extension ReviewsDashboardCard {
         }
     }
 
-    func authorText(author: String, productName: String) -> some View {
-        if author.isNotEmpty {
-            return Text(String.localizedStringWithFormat(Localization.completeAuthorText, author, productName))
-        } else {
-            return Text(String.localizedStringWithFormat(Localization.incompleteAuthorText, productName))
-        }
-    }
-
-    func reviewText(text: String, shouldDisplayStatus: Bool) -> some View {
+    func reviewText(text: String,
+                    pendingText: String,
+                    divider: String,
+                    textColor: UIColor,
+                    accentColor: UIColor,
+                    shouldDisplayStatus: Bool) -> some View {
         if shouldDisplayStatus {
-            return Text(Localization.pendingReview).foregroundColor(Color(uiColor: .wooOrange)) +
-                Text(" • " + text)
+            return Text(pendingText).foregroundColor(Color(uiColor: accentColor)) +
+            Text(divider + text).foregroundColor(Color(uiColor: textColor))
```

Comment:
here I wanted to use .foregroundStyle but Xcode said it's only available on iOS 17 above 🤔 
![Screenshot 2024-05-17 at 14 17 24](https://github.com/woocommerce/woocommerce-ios/assets/266376/73e2370e-6fc8-4242-8455-50e4d86c08a1)

Reply:
I followed the guide here:

https://www.hackingwithswift.com/quick-start/swiftui/how-to-combine-text-views-together

Reply:
I ended up updating to use AttributedString instead just to avoid using the deprecated `.foregroundColor()`. Thanks!

https://github.com/woocommerce/woocommerce-ios/pull/12752/commits/ae19a09781a0e1e40f2a672d0a85c6792f50a3b5